exercises/hamming/example.hs

```diff
@@ -1,4 +1,4 @@
-module DNA (hammingDistance) where
+module Hamming (hammingDistance) where
```

Comment:
going to say yes to this. see how https://hackage.haskell.org/package/containers-0.5.7.1/docs/Data-Set.html and https://hackage.haskell.org/package/containers-0.5.7.1/docs/Data-Map-Lazy.html just have a `size` instead of, say, `setSize` or `mapSize` or something. Coming soon.